rwengine/src/script/ScriptFunctions.hpp

```diff
@@ -6,12 +6,15 @@
 #include <engine/GameData.hpp>
 #include <objects/GameObject.hpp>
 #include <script/ScriptTypes.hpp>
+#include <script/ScriptMachine.hpp>
 
 /**
  * Implementaitions for common functions likely to be shared
  * among many script modules and opcodes.
  */
 namespace script {
+const ScriptVec3 kSpawnOffset = ScriptVec3(0.f, 0.f, 1.f);
```

Comment:
The typical spawn offset we used back in the day was 0.981 (which would be 1 frame worth of gravity probably)
I remember because we wrote a physics test at school and I used g = 0.981 m/(s*s) because I somehow mixed it up.

Maybe that was the proper offset? I must have heard of it from gtaforums or the MTA forums.